documentation/modules/managing/con-namespace-deletion-recovery.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assembly:
+//
+// assembly-cluster-recovery-volume.adoc
+
+[id="namespace-deletion_{context}"]
+= Namespace deletion
+
+Recovery from namespace deletion is possible because of the relationship between persistent volumes and namespaces.
+A `PersistentVolume` (PV) is a storage resource that lives outside of a namespace.
+A PV is mounted into a Kafka pod using a `PersistentVolumeClaim` (PVC), which lives inside a namespace.
+
+If a namespace is deleted unintentionally, you can recover it from a PV as long as it is still present.
+
+A reclaim policy for a PV tells a cluster how to act when a namespace is deleted.
+If the reclaim policy is set as _Delete_, PVs are deleted when PVCs are deleted with a namespace.
+
+To ensure that PVs are not deleted when a namespace is deleted, the policy must be reset from _Delete_ to _Retain_.
+You can also configure the `StorageClass` property used for dynamic volume allocation so that it creates the volumes with the appropriate reclaim policy.
+
+NOTE: If you are using _Retain_ as the reclaim policy, but you want to delete an entire cluster, you need to delete the PVs manually.
+Otherwise they will not be deleted, and may cause unnecessary expenditure.
```

Comment:
Expenditure of memory, resources, or something else?

Reply:
...may cause unnecessary expenditure on resources